javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,87 @@
+//FreeCOdeCamp Exercise link
+//https://www.freecodecamp.org/myselfankitaa
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1988;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2045;
+const dogAge = dogYearFuture - dogYearOfBirth;
+let dogYear = dogAge * 7;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogAge + " human years old in " + dogYearFuture
+  );
+}
+
+//Housey pricey (A house price estimator)
+const peterHouseWidth = 8;
+const peterHouseDepth = 10;
+const peterHouseheight = 10;
+const peterGardenSizeInM2 = 100;
+const peterHouseActualPrice = 2500000;
+let volumeInMeters = peterHouseheight * peterHouseDepth * peterHouseWidth;
+let housePrice = volumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
```

Comment:
Great how you have explicitly separated each value and given it it's own variable!

I would even recommend explicitly giving the values 2.5, 1000 and 300 variable names. This makes it easier for future reference to understand what these numbers represent!

Reply:
As for formula for the house price, it was given as it is in the home-work. So I really don't know what these numbers represents. I am just assuming these formula as a constant to calculate the house price by putting values of defined variables.
If it is known, what that number represents, I will make the variables accordingly.